webscrambles/src/main/resources/tnoodle_resources/i18n/da.yml

```diff
@@ -20,6 +20,8 @@ da:
         result: Resultat
         #original_hash: 1917f4f
         competition: Konkurrence
+        #original_hash: ca817e3
+        registrantId: Registrerings-ID
```

Comment:
Could we actually change this to "Tilmeldings-ID"? I realise this would be more consistent with how registration has been translated elsewhere on the website.

Reply:
Fixed. Sorry for the delay